client/src/MoreInfoPage.js

```diff
@@ -0,0 +1,103 @@
+import { BsFillHeartFill } from 'react-icons/bs';
+import { useLocation } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+
+export default function  MoreInfoPage() {
+  return (
+    <MoreInfo/>
+  )
+}
+
+function MoreInfo() {
+  const location = useLocation();
+  const navigate = useNavigate();
+  const readBookObject = location.state;
+
+    async function addBook() {
+    let book = { title: 'Title Unknown', author: 'Author Unknown', isbn: 'Not Found', rating: 0, image: "https://blog.springshare.com/wp-content/uploads/2010/02/nc-md.gif", price: 19.99, quantity: 1 }
+    if (readBookObject.volumeInfo.title) {
```

Comment:
You could destructure `readBookObject.volumeInfo` to make this code more succinct.